hw/ip/prim/lint/prim_lfsr.waiver

```diff
@@ -5,5 +5,5 @@
 # waiver file for prim_lfsr
 
 # TODO: this lint waiver may not be necessary in future AscentLint versions. check and remove if possible.
-waive -rules {LOOP_VAR_OP} -location {prim_lfsr.sv} -msg {Loop variable 'k' is in arithmetic expression '(k + shift) % gen_out_non_linear.NumSboxes' with non-constant terms} \
+waive -rules {LOOP_VAR_OP} -location {prim_lfsr.sv} -regexp {Loop variable 'k' is in arithmetic expression '\(k \+ shift\) % (gen_out_non_linear.|)NumSboxes' with non-constant terms} \
```

Comment:
Definitely not a problem, but I'm surprised this syntax is required by AscentLint. Don't they support `(optional_thing)?` ?

Reply:
Ah, now I understand, thanks. This seems indeed to be working and I will do it this way next time.